### DISCUSSION TITLE
能放一个可以编辑的api base在config里吗？

### DISCUSSION THREAD

**Discussion by inbengable1** [binary-husky/gpt_academic#535] [Q&A]
能放一个可以编辑的api base在config里吗？有些国内用户喜欢由中转网站使用api，就是在项目里调用中转网站的api，中转网站再调用openai的api。对于我这样的小用户，这比搞一个虚拟信用卡便宜很多。但是这样得把openai的apibase改成中转网站的。我记得前几个版本是有这个功能的，现在好像删掉了。

**Comment by binary-husky** [comment]
@inbengable1 目前已经支持api2d的接口，其他接口也欢迎PR

**Reply by MZGuo111** [reply]
作者您好！请问api2d的gpt-4模型如何更换呢？Available模型中好像没有gpt-4的版本
